### DISCUSSION TITLE
Pre-trigger recording -- retaining a few seconds before the trigger?

### DISCUSSION THREAD

**Discussion by dsjstc** [roflcoopter/viseron#680] [Q&A]
I'd like to start recording about 5 seconds before the trigger event.  That's because at 1FPS, the trigger event usually happens well after an object has entered the field of view.

I'm guessing that this would be an NVR parameter (the "recorder" is in the NVR component right?), but the component page for NVR doesn't have any configurables.

Is this a feature?

**Comment by roflcoopter** [comment]
Yes you can do that using the `lookback` option for `recorder` in your camera config.

The recorder is not under the `nvr` component since there are specific options for it tied to the  component providing the camera

**Reply by coderunway** [reply]
Thanks this helped me.
